package.json

```diff
@@ -36,6 +36,7 @@
     "prop-types": "^15.5.10",
     "react-lifecycles-compat": "^1.0.2",
     "react-native-drawer-layout-polyfill": "^1.3.2",
+    "react-native-gesture-handler": "^1.0.0-alpha.41",
```

Comment:
we should make this a peer dependency

Reply:
Is there a good way to handle the [different version requirements](https://github.com/kmagiera/react-native-gesture-handler#supported-configurations) for different versions of react-native, or would we just set the loosest spec here (`^1.0.0-alpha.29`) and document the required versions? Should we also make the react-native requirement stricter (`>=0.47`)? Or is there a way I'm unaware of to tie two peerDep versions together?

Tangentially related: we may also want to consider adding support for using the `GestureHandler` included in Expo.

Reply:
@brettdh - sorry I haven't had time to look back at this until now. I think we should only release this when rngh has left alpha, at which point we can assume that it will follow semver and we can support major version 1

Reply:
@brentvatne Makes sense to me, though of course I'll be sad for the delay. What about the Expo considerations? Or would it not matter, since [the `expo` package depends on `react-native-gesture-handler`](https://github.com/expo/expo-sdk/blob/master/package.json#L59) anyway, thereby satisfying the peer dependency?

Reply:
> Or would it not matter, since the expo package depends on react-native-gesture-handler anyway, thereby satisfying the peer dependency?

precisely.

> Makes sense to me, though of course I'll be sad for the delay.

I suppose what we could do here is specify a peerDependency that is satisfied by both pre-release and uh, release versions (what's the proper term for this anyhow? :P) I don't anticipate that the API for drawer will change significantly, so it should be safe.

Reply:
Updated, rebased, pushed.